### ISSUE TITLE
The 'queryset' attribute on HaystackViewSet is apparently ignored in get_queryset

### ISSUE THREAD

**Issue by lavoiedn** [inonit/drf-haystack#90]
Trying to set the HaystackViewSet's `queryset` attribute does not work with the current `get_queryset` implementation.

This forces the ViewSet to override `get_queryset` to properly change its SearchQuerySet.

For instance:

```python
from drf_haystack.viewsets import HaystackViewSet

class ProjectSearchViewSet(HaystackViewSet):
    index_models = [Project]
    queryset = SearchQuerySet().filter(status=1)
    serializer_class = ProjectSearchSerializer
```

Will not alter the resulting queryset at all.

Currently, it would have to be implemented as such:

```python
from drf_haystack.viewsets import HaystackViewSet

class ProjectSearchViewSet(HaystackViewSet):
    index_models = [Project]
    serializer_class = ProjectSearchSerializer

    def get_queryset(self, index_models=[]):
      return super(ProjectSearchViewSet, self).get_queryset(index_models).filter(status=1)
```

I tried fixing it to the best of my knowledge with [this commit](https://github.com/DesjardinsLab/drf-haystack/commit/1e96353cbff6e727488c1754e59aba9e3fe568dd#diff-c46f1150d4768d66f5a64c088b76900a).

The `if` block in `get_queryset` would always enter the `else` clause since the `if` clause's condition requires `self.queryset = None` and `self.object_class = type(None)` (Which would cause an exception when calling None.all()).

I was not sure of the intended behaviour, but I assumed that the `not` was omitted by accident.

I the fix is judged adequate, I can submit a pull request.

**Comment by rhblind** [comment]
Whoops, yep seems like a [miss](https://github.com/DesjardinsLab/drf-haystack/commit/1e96353cbff6e727488c1754e59aba9e3fe568dd)!

A PR is most welcome =)

**Comment by rhblind** [comment]
Fixed in #91
